ui/src/app/modules/settings/settings.component.ts

```diff
@@ -131,6 +132,7 @@ export class SettingsComponent implements OnInit {
           this.$settings.env.runningInPackageMode
         ) {
           this.showAvahiMdnsOption = true;
+          this.showResolvedMdnsOption = true;
```

Comment:
Please only show the resolved option if the homebridge version is `1.6.0-beta.0` (the planned homebridge release to include hap-nodejs 0.11.0) or higher, as above.